src/main/steps/state-sequence.test.ts

```diff
@@ -34,4 +40,38 @@ describe('StateSequence', () => {
     const mockUserCase = { state: State.AwaitingPronouncement };
     expect(currentStateFn(mockUserCase.state).state()).toEqual(State.AwaitingPronouncement);
   });
+
+  test('preSubmittedStatePrioritySequence should be appropriate', async () => {
+    expect(preSubmittedStatePrioritySequence).toHaveLength(8);
```

Comment:
is the number of states important?

Reply:
The length isn't actually important, I added it to make sure that whoever makes a change to the state sequence understands the consequences. But maybe that's not important?